lib/Platform/Intl/PlatformIntlApple.mm

```diff
@@ -113,14 +131,15 @@
 struct LocaleMatch {
   std::u16string locale;
   std::u16string extension;
+  std::u16string matchedLocale;
 };
 LocaleMatch lookupMatcher(
-    const std::vector<std::u16string> &requestedLocales,
+    const vm::CallResult<std::vector<std::u16string>> &requestedLocales,
```

Comment:
Does it have to be `CallResult`? Feels a little odd that `CallResult` is passed as an input to a function.